files/lib/data/faq/Question.class.php

```diff
@@ -7,14 +7,15 @@
 use wcf\data\faq\category\FaqCategory;
 use wcf\data\user\User;
 use wcf\data\DatabaseObject;
+use wcf\data\search\ICustomIconSearchResultObject;
 use wcf\data\search\ISearchResultObject;
 use wcf\page\FaqQuestionPage;
 use wcf\system\html\output\HtmlOutputProcessor;
 use wcf\system\request\IRouteController;
 use wcf\system\request\LinkHandler;
 use wcf\system\WCF;
 
-class Question extends DatabaseObject implements IRouteController, ISearchResultObject
+class Question extends DatabaseObject implements ICustomIconSearchResultObject, IRouteController, ISearchResultObject
```

Comment:
`ICustomIconSearchResultObject` extends `ISearchResultObject`, so there's no need to implement `ISearchResult` if `ICustomIconSearchResultObject` is implemented 🙂